src/components/ItemList/ItemList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface PropsType<T> {
+  items: T[];
+  renderer: (item: T, index: number, arr: T[]) => React.ReactNode;
+  className?: string;
+}
+
+export default function ListView<T extends { id: number }>(
```

Comment:
Please rename the component to match the folder/filename too

```suggestion
export default function ItemList<T extends { id: number }>(
```

(technically this won't break anything since anybody can rename a default export whenever it's imported, but better practice to keep them consistent)

Reply:
Fixed! [4cb382a
](https://github.com/Code-the-Dream-School/ee-prac-team2-front/pull/21/commits/4cb382a2718c3ee68ce03b1834ecd8f181c3b202)